### DISCUSSION TITLE
Best way to add channels when they aren't listed in the EPG?

### DISCUSSION THREAD

**Discussion by nabeelr** [iptv-org/community#574] [Q&A] [answered]
What's the best way to create GH issues to add new channels?

I've had several rejected because they weren't in the EPG, or because I couldn't answer some of the questions to add it to the EPG or the iptv database.

There's a bunch of pluto tv channels that could be added, but most of my attempts have failed.

Suggestions? What's the best way to file these GH issues?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
Checkout the [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Reply by nabeelr** [reply]
yeah, I did that and I followed it as close as I could, but either the guide is providing incorrect steps, I'm being rejected by mistake, or there's some misunderstanding between what the guide means to say and what I understand it to say.

**Comment by PopeyeTheSai10r** [comment]
Can you link me to some of your contributions that got rejected?

**Reply by nabeelr** [reply]
I submitted two, the first was ultimately accepted, the second was rejected:
https://github.com/iptv-org/database/issues/4599
https://github.com/iptv-org/database/issues/4600

**Reply by PopeyeTheSai10r** [reply]
Looks like [BellezaEmporium](https://github.com/BellezaEmporium) responded and provided some clarity.
You will need to split those request into two one for each country.
